src/linux/backend-hid.cpp

```diff
@@ -687,6 +687,9 @@ namespace librealsense
             auto raw_data_size = channel_size*hid_buf_len;
 
             std::vector<uint8_t> raw_data(raw_data_size);
+            
+            if( _fd < 0 )
```

Comment:
Redundant, this is checked on line 680.
Linux API declares this parameter as an int, we won't change the type to unsigned. Negative means `open` failed.

Reply:
This should be marked as not an error in Coverity.

Reply:
The lines were removed.